gpkit/nomials.py

```diff
@@ -147,14 +155,25 @@ def value(self):
     def to(self, arg):
         return Signomial(self.exps, self.cs.to(arg).tolist())
 
-    def diff(self, var):
-        if var in self.varkeys:
-            var = self.varkeys[var]
-        elif isinstance(var, Monomial):
-            vks = list(var.exp)
+    def diff(self, wrt):
+        """Derivative of this with respect to a Variable
+
+        Arguments
+        ---------
+        wrt (Variable):
+            Variable to take derivative with respect to
+
+        Returns
+        -------
+        Signomial (or Posynomial or Monomial)
+        """
+        if wrt in self.varkeys:
+            wrt = self.varkeys[wrt]
+        elif isinstance(wrt, Monomial):
+            vks = list(wrt.exp)
```

Comment:
By the way, do we actually support differentiation with respect to a `Monomial`? How exactly is that defined? (It's currently missing from the docs). It seems like the current implementation assumes that the `Monomial` has only one `VarKey`?